### ISSUE TITLE
Unable to create a seed object via python

### ISSUE THREAD

**Issue by rrgirish** [chrismattmann/nutch-python#5]
When i run the python code:

```
def get_seed(seedListData=('http://www.zidaho.com','http://www.iguntrade.com')):
    sc = get_seed_client()
    return sc.create('test_seed', seedListData)
```

it fails with the below being displayed in the stdout:

nutch.py: POST Request data: {'id': '12345', 'seedUrls': [{'url': 'http://www.zidaho.com', 'seedList': None, 'id': 0}, {'url': 'http://www.iguntrade.com', 'seedList': None, 'id': 1}], 'name': 'test_seed'}
nutch.py: POST Request headers: {'Accept': 'application/json'}
nutch.py: Response headers: {'date': 'Sat, 10 Oct 2015 18:00:58 GMT', 'content-length': '0', 'server': 'Jetty(8.1.15.v20140411)'}
nutch.py: Response status: 500
Traceback (most recent call last):
  File "nutchPythonCrawl.py", line 79, in <module>
    test_crawl_client()
  File "nutchPythonCrawl.py", line 73, in test_crawl_client
    cc = get_crawl_client()
  File "nutchPythonCrawl.py", line 30, in get_crawl_client
    seed = get_seed()
  File "nutchPythonCrawl.py", line 21, in get_seed
    return sc.create('test_seed', seedListData)
  File "/Users/rrgirish/anaconda/lib/python2.7/site-packages/nutch/nutch.py", line 433, in create
    seedPath = self.server.call('post', "/seed/create", seedListData, JsonAcceptHeader, forceText=True)
  File "/Users/rrgirish/anaconda/lib/python2.7/site-packages/nutch/nutch.py", line 160, in call
    raise error

**Comment by Rohnak** [comment]
When I run:
seed_urls = ('http://www.google.com','http://www.bing.com')
sc = nt.Seeds()
sc.create('seedst',seed_urls)
I get:
nutch.py: POST Endpoint: /seed/create
nutch.py: POST Request data: {'id': '12345', 'seedUrls': [{'url': 'http://www.google.com', 'seedList': None, 'id': 0}, {'url': 'http://www.bing.com', 'seedList': None, 'id': 1}], 'name': 'seedst'}
nutch.py: POST Request headers: {'Accept': 'application/json'}
nutch.py: Response headers: {'date': 'Mon, 12 Oct 2015 01:02:49 GMT', 'content-length': '0', 'server': 'Jetty(8.1.15.v20140411)'}
nutch.py: Response status: 406
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ronak/anaconda/lib/python2.7/site-packages/nutch/nutch.py", line 433, in create
    seedPath = self.server.call('post', "/seed/create", seedListData, JsonAcceptHeader, forceText=True)
  File "/home/ronak/anaconda/lib/python2.7/site-packages/nutch/nutch.py", line 160, in call
    raise error
nutch.nutch.NutchException: Unexpected server response: 406

**Comment by chrismattmann** [comment]
I can confirm this behavior:

```
[chipotle:~/git] mattmann% python2.7
Python 2.7.8 (default, Sep 27 2014, 11:46:04) 
[GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from nutch.nutch import Nutch
>>> from nutch.nutch import SeedClient
>>> from nutch.nutch import Server
>>> import nutch
>>> sv=Server('http://localhost:8081')
>>> sc=SeedClient(sv)
>>> seed_urls=['http://espn.go.com', 'http://www.espn.com']
>>> sd = sc.create('espn-seed', seed_urls)
nutch.py: POST Endpoint: /seed/create
nutch.py: POST Request data: {'id': '12345', 'seedUrls': [{'url': 'http://espn.go.com', 'id': 0}, {'url': 'http://www.espn.com', 'id': 1}], 'name': 'espn-seed'}
nutch.py: POST Request headers: {'Accept': 'application/json'}
nutch.py: Response headers: {'date': 'Tue, 13 Oct 2015 01:48:37 GMT', 'content-length': '0', 'server': 'Jetty(8.1.15.v20140411)'}
nutch.py: Response status: 406
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build/bdist.macosx-10.9-x86_64/egg/nutch/nutch.py", line 433, in create
  File "build/bdist.macosx-10.9-x86_64/egg/nutch/nutch.py", line 160, in call
nutch.nutch.NutchException: Unexpected server response: 406
```

**Comment by chrismattmann** [comment]
I can confirm this is fixed in #7 

```
[chipotle:~/git] mattmann% python2.7
Python 2.7.8 (default, Sep 27 2014, 11:46:04) 
[GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from nutch.nutch import Nutch
>>> from nutch.nutch import SeedClient
>>> from nutch.nutch import Server
>>> import nutch
>>> 
>>> sv=Server('http://localhost:8081')
>>> sc=SeedClient(sv)
>>> seed_urls=('http://espn.go.com','http://www.espn.com')
>>> sd= sc.create('espn-seed',seed_urls) 
nutch.py: POST Endpoint: /seed/create
nutch.py: POST Request data: {'id': '12345', 'seedUrls': [{'url': 'http://espn.go.com', 'id': 0}, {'url': 'http://www.espn.com', 'id': 1}], 'name': 'espn-seed'}
nutch.py: POST Request headers: {'Accept': 'text/plain'}
nutch.py: Response headers: {'date': 'Tue, 13 Oct 2015 01:52:41 GMT', 'content-length': '64', 'content-type': 'text/plain', 'server': 'Jetty(8.1.15.v20140411)'}
nutch.py: Response status: 200
nutch.py: Response text: /var/folders/05/5qw82z2d77q16fhxxhwt22tr0000gq/T/1444701161927-0
>>> 
```

Thanks @thammegowda !
